src/index.ts

```diff
@@ -7,15 +7,22 @@ import { GraphQLField, GraphQLType } from "./types";
 class GraphqlJsonToSdl extends Command {
   static description = "Converts a JSON GraphQL schema to GraphQL SDL.";
 
-  static examples = ["$ graphql-json-to-sdl ./schema.json ./schema.graphql"];
+  static examples = [
+    "$ graphql-json-to-sdl ./schema.json ./schema.graphql",
+    "$ graphql-json-to-sdl -r ./schema.json ./schema.graphql",
```

Comment:
```suggestion
```

Looks like the example for the `-r` flag accidentally got left behind here.

Reply:
fixed